### PR TITLE
Adding exception info to kickstarter in others.yml

### PR DESCRIPTION
**Pull request by jgfam69qy2**
Info received as response from kickstarter's support team in a ticket.
image

### CODE REVIEW

File: _data/other.yml [hunk 1]
```diff
@@ -373,6 +373,7 @@ websites:
       - phone
       - totp
     doc: https://help.kickstarter.com/hc/en-us/articles/115005127094
+    exception: "Currently requires a verified SMS Number as fallback for Software 2FA."
```
Comment (ApCoder123) [inline _data/other.yml]:
```suggestion
    exception: "Software 2FA requires setting up SMS 2FA as a fallback."
```

I think that this reads slightly better?

It sounds more similar to existing exception text as well: E.g. for Dropbox "Hardware 2FA requires enabling Software or SMS 2FA as a back-up."

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

**Review by ApCoder123 — APPROVED** [review]

File: _data/other.yml (hunk 1 above)
Comment (jgfam69qy2) [reply _data/other.yml]: Thank you. 
It is much clearer that way.

**Review by phallobst — COMMENTED** [review]
Do they require to actually enable SMS 2FA, or just to supply a phone number? Or is SMS 2FA available anyway as soon as the phone number is given?

**Comment by jgfam69qy2** [comment]
> Do they require to actually enable SMS 2FA, or just to supply a phone number? Or is SMS 2FA available anyway as soon as the phone number is given?

I went through the process
There seems to be no option to use ONLY SMS or TOTP as it verifies both.

Effectively both 2nd factors end up enabled (see "Send code to my Phone" at the bottom of the snapshot).

image

Furthermore, SMS cannot be disabled, only updated, and disabling OTP disables 2FA altoghether

image

**Review by phallobst — APPROVED** [review]

**Pull request merged**